rubygem-jenkins_api_client/rubygem-jenkins_api_client.spec

```diff
@@ -0,0 +1,106 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+%global gem_name jenkins_api_client
+
+Summary: Jenkins Api client with features focused on automating Job configuration programaticaly
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.4.1
+Release: 1%{?dist}
+Group: Developmanet/Languages
+License: MIT
+URL: https://github.com/arangamani/jenkins_api_client
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ruby}ruby
+
+%if "%{?scl_ruby}" == "ruby193" || (0%{?el6} && 0%{!?scl:1})
+Requires: %{?scl_prefix_ruby}ruby(abi)
+BuildRequires: %{?scl_prefix_ruby}ruby(abi)
+%else
+Requires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+%endif
+
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby
+
+Requires: %{?scl_prefix_ruby}rubygem(json)
+Requires: %{?scl_prefix_ruby}rubygem(mixlib-shellout) >= 1.1.0
+Requires: %{?scl_prefix_ruby}rubygem(terminal-table) >= 1.4.0
+Requires: %{?scl_prefix_ruby}rubygem(thor) >= 0.16.0
+Requires: %{?scl_prefix_ruby}rubygem(nokogiri) >= 1.6.0
+Requires: %{?scl_prefix_ruby}rubygem(nokogiri) < 2.0.0
+
+BuildRequires: %{?scl_prefix_ruby}rubygem(json)
+BuildRequires: %{?scl_prefix_ruby}rubygem(mixlib-shellout) >= 1.1.0
+BuildRequires: %{?scl_prefix_ruby}rubygem(terminal-table) >= 1.4.0
```

Comment:
mixlib-shellout and terminal-table will need to use `%{?scl_prefix}` since we build them into the `tfm-` namespace instead of the ruby scl namespace. Same is true in the Requires section above.